com.unity.multiplayer.mlapi/Tests/Runtime/com.unity.multiplayer.mlapi.runtimetests.asmdef

```diff
@@ -5,11 +5,21 @@
         "Unity.Multiplayer.MLAPI.Runtime",
         "Unity.Multiplayer.NetworkProfiler.Runtime",
         "Unity.Multiplayer.NetStats",
-        "Unity.Multiplayer.MLAPI.Editor"
-    ],
-    "optionalUnityReferences": [
-        "TestAssemblies"
+        "Unity.Multiplayer.MLAPI.Editor",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner"
     ],
     "includePlatforms": [],
-    "excludePlatforms": []
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": true,
+    "precompiledReferences": [
+        "nunit.framework.dll"
+    ],
+    "autoReferenced": false,
+    "defineConstraints": [
+        "UNITY_INCLUDE_TESTS"
+    ],
+    "versionDefines": [],
+    "noEngineReferences": false
```

Comment:
nothing to worry yet but just a heads-up: we tend to keep `.asmdef` files super clean and minimal and some of these diff changes are not actually needed.

for instance, this (previous diff):
```diff
- "optionalUnityReferences": [
-     "TestAssemblies"
- ]
```

is equivalent to this:
```diff
+     "UnityEngine.TestRunner",
+     "UnityEditor.TestRunner"
+ ],
+ "precompiledReferences": [
+     "nunit.framework.dll"
+ ],
+ "defineConstraints": [
+     "UNITY_INCLUDE_TESTS"
+ ]
...
```

the main difference is that we don't touch/interfere with any magic happening behind the scenes caused by "TestAssemblies" :)

Reply:
I also see some GUID refs in MLAPI.Runtime asmdef above

```
    "references": [
        "GUID:3450dc9e228065a499e6e09dee931704",
        "GUID:b1cd7326e664a434ab35daa802773c7f"
    ],
```

but anyways, I'll probably talk more about asmdef stuff when a PR that targets merging `experimental/netstats-dispatcher` into `develop` comes alive :P

Reply:
Yup, this is what Unity does when you modify the asmdef in-editor though.